modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -6,7 +6,23 @@ endif::[]
 The Mule Maven plugin enables you to integrate the packaging, testing, and deployment of your Mule applications with your Maven lifecycle. +
 See xref:package-a-mule-application.adoc[Package a Mule Application] for packaging instructions.
 
-With the plugin, you can deploy your Mule application to different deployment targets, such as standalone Mule instances, clustered instances, and CloudHub.
+You can automate the deployment of your Mule application to any of the application deployment targets (CloudHub, Runtime Fabric, or On-Premises). +
+The Mule Maven plugin enables you to deploy using five different deployment strategies:
+
```

Comment:
Do we really need the descriptions of these list items?
Also, only the first mentions automation, but I thought that the previous text implies that they are all automated?
Clarify.

Reply:
Reconsider including the xrefs, since each topic is covered within the topic set itself.